src/Twig/Extension/SonataTranslationExtension.php

```diff
@@ -26,6 +28,12 @@ class SonataTranslationExtension extends \Twig_Extension
     protected $translatableChecker;
 
     /**
+     * @var LocaleBundleInterface
+     */
+    protected $localeBundle;
```

Comment:
Using `protected` everywhere was a mistake on our part. We are trying to move to `private` as much as we can for new fields:

```suggestion
    private $localeBundle;
```